### DISCUSSION TITLE
Randomly Insert Rare items - Replace Util

### DISCUSSION THREAD

**Discussion by ultraviolet007rainbow** [nftchef/art-engine#195] [Help + Q&A]
Hi, 

Can you please post a sample JSON file of what this should look like exactly, do we not include the DNA as there is no DNA?
Date? Imagehash? etc.?
Does it have to match all the other JSON files as far as all the Metadata because it can't since these 1/1s.
Please clarify.

Thanks :)

**Comment by nftchef** [comment]
Updated documentation for the Replace util with example metadata is now here
https://generator.nftchef.dev/utils/replace-for-rares

tl;dr, `dna` is not required. Only 'standard' metadata is needed, like `image`, `name`, and `attributes` if they are being used.
